core/src/bmi.jl

```diff
@@ -248,7 +248,7 @@ function create_callbacks(
             save_subgrid_level,
             saved_subgrid_level;
             saveat,
-            save_start = false,
+            save_start = true,
```

Comment:
@Huite I'm now saving initial subgrid levels as well, just like the regular basin.arrow. I assume you copied this from flow, which is undefined at t0 so we don't save it.

Reply:
Yes, this sounds fine.